diff --git a/node_modules/@baronha/react-native-photo-editor/android/src/main/java/com/reactnativephotoeditor/PhotoEditorModule.kt b/node_modules/@baronha/react-native-photo-editor/android/src/main/java/com/reactnativephotoeditor/PhotoEditorModule.kt
index c1f46c9..8ae5b7d 100644
--- a/node_modules/@baronha/react-native-photo-editor/android/src/main/java/com/reactnativephotoeditor/PhotoEditorModule.kt
+++ b/node_modules/@baronha/react-native-photo-editor/android/src/main/java/com/reactnativephotoeditor/PhotoEditorModule.kt
@@ -21,7 +21,7 @@ class PhotoEditorModule(reactContext: ReactApplicationContext) : ReactContextBas
   @ReactMethod
   fun open(options: ReadableMap?, promise: Promise): Unit {
     this.promise = promise
-    val activity = currentActivity
+    val activity = reactApplicationContext.currentActivity
     if (activity == null) {
       promise.reject("ACTIVITY_DOES_NOT_EXIST", "Activity doesn't exist");
       return;
diff --git a/node_modules/@baronha/react-native-photo-editor/react-native-photo-editor.podspec b/node_modules/@baronha/react-native-photo-editor/react-native-photo-editor.podspec
index 61bc5bc..5d29ac1 100644
--- a/node_modules/@baronha/react-native-photo-editor/react-native-photo-editor.podspec
+++ b/node_modules/@baronha/react-native-photo-editor/react-native-photo-editor.podspec
@@ -17,8 +17,8 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm,swift}"
   
   s.dependency "React-Core"
-  s.dependency "SDWebImage", "~> 5.11.1"
-  s.dependency 'SDWebImageWebPCoder', '~> 0.8.4'
+  s.dependency "SDWebImage", ">= 5.11.1"
+  s.dependency 'SDWebImageWebPCoder', '>= 0.8.4'
 
   s.subspec 'ZLImageEditor' do |zl|
     zl.name             = "ZLImageEditor"
